web_field_tooltip/models/ir_model_fields_tooltip.py

```diff
@@ -34,7 +34,7 @@ class IrModelFieldsTooltip(models.Model):
         help="Set active to false to hide the Tooltip without removing it.",
     )
     field_name = fields.Char(related="field_id.name")
-    tooltip_text = fields.Html(required=True)
+    tooltip_text = fields.Text(required=True, translate=True)
```

Comment:
maybe this change requires a migration script for users coming from V16.

not a blocking point, though. Can be done in another PR

Reply:
I think now is the right time for this, you just need to use https://github.com/odoo/odoo/blob/d1f20d6b71203e6bb4b428ae1bcbb0b76e739f51/odoo/tools/mail.py#L350

Reply:
Migration script added